app/__tests__/services/media.test.ts

```
@@ -86,106 +236,713 @@ describe('MediaManager', () => {
-    const result2 = mediaManager.toggleVideo();
-    expect(result2).toBe(true);
+      // Test with supported browser (mock has setSinkId)
+      const result1 = await mediaManager.switchAudioOutputDevice(
+        'output1',
+        mockAudioElement as unknown as HTMLMediaElement
+      );
+      expect(result1).toBe(true);
+      expect(mockAudioElement.setSinkId?.mock.calls[0][0]).toBe('output1');
+
+      // Test with unsupported browser (remove setSinkId)
+      delete mockAudioElement.setSinkId;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)